psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -1996,4 +2046,1146 @@ void MintsHelper::play()
 {
 }
 
-}  // namespace psi
+/* 1st and 2nd derivatives of OEI in AO basis  */
+
+std::vector<SharedMatrix> MintsHelper::ao_overlap_kinetic_deriv1_helper(const std::string & type, int atom)
+{
+        char lbl[32];
+        char lbl1[32];
+        char ** cartcomp;
+        cartcomp = (char **) malloc (3 * sizeof(char *));
+        cartcomp[0] = strdup("X");
+        cartcomp[1] = strdup("Y");
+        cartcomp[2] = strdup("Z");
```

Comment:
Great place to use `std:vector<std::string>`.

Reply:
Done!